metadata-ingestion/src/datahub/emitter/request_helper.py

```diff
@@ -31,3 +41,97 @@ def make_curl_command(
 
     fragments.append(url)
     return shlex.join(fragments)
+
+
+@dataclass
+class OpenApiRequest:
```

Comment:
Given this is factory class for OpenAPI v3 requests, wondering if we should emphasise that in the class name; _eg_ `OpenApiV3Request`.